code/controllers/subsystem.dm

```diff
@@ -255,14 +255,18 @@
 /datum/controller/subsystem/proc/OnConfigLoad()
 
 //used to initialize the subsystem AFTER the map has loaded
-/datum/controller/subsystem/Initialize(start_timeofday)
+/datum/controller/subsystem/Initialize()
 	initialized = TRUE
-	SEND_SIGNAL(src, COMSIG_SUBSYSTEM_POST_INITIALIZE, start_timeofday)
-	var/time = (REALTIMEOFDAY - start_timeofday) / 10
-	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
+	SEND_SIGNAL(src, COMSIG_SUBSYSTEM_POST_INITIALIZE)
```

Comment:
nothing relied on this giving start_timeofday

Reply:
edit the signal definition, you'll mislead people like this

Reply:
done